src/app/designationEditor/designationEditor.component.js

```diff
@@ -341,6 +342,21 @@ class DesignationEditorController {
       this.carouselLoaded = true
     }
   }
+
+  getDoneEditingUrl () {
+    const vanityPath = this.designationContent['sling:vanityPath']
+    const cacheBust = '?doneEditing'
+
+    if (vanityPath) {
+      if (isArray(vanityPath)) {
```

Comment:
I believe you could use `Array.isArray` here instead of needing a lodash import.

Reply:
There also seems to be `angular.isArray` in use in other parts of the project.

Reply:
I mainly chose lodash for consistency, but the project itself is not consistent.